students/guillaume/session03/slicing_lab.py

```diff
@@ -46,15 +42,19 @@ def third(seq):
     l = len(seq)
     if l % 3 == 0:
         third = l // 3
-        new_seq = seq[third:2 * third] + seq[2 * third: ] + seq[0:third]
+        new_seq = seq[third:2 * third] + seq[2 * third:] + seq[0:third]
         return new_seq
     print('len of the sequence is not a multiple of 3')
     return None
 
+
 if __name__ == '__main__':
 
-    functions = [exch_first_last, every_other_rem, first_last_in_bet, reverse_slic, third]
-    test_lst = ['', 'tesu', 'ab',''.join(map(str,list(range(10)))), [1,2,3], list(range(24)), list(range(9)),['1third', '2third', '3third'], []]
+    functions = [exch_first_last, every_other_rem,
+                 first_last_in_bet, reverse_slic, third]
+    test_lst = ['', 'tesu', 'ab', ''.join(map(str, list(range(10)))),
```

Comment:
this is just a style issue, but I think it would be clearer if you put each on one line:

```
test_lst = ['',
                 'tesu',
                 'ab',
                 ''.join(map(str, list(range(10)))),[1,2,3],
                 list(range(24)),
                 list(range(9)),
                 ['1third', '2third', '3third'],
                 [],
                 ]
```
It took me a while to see which belonged with which.